packages/xai/src/responses/xai-responses-options.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from 'zod/v4';
+
+export type XaiResponsesModelId =
+  | 'grok-4'
+  | 'grok-4-fast'
+  | 'grok-4-fast-non-reasoning'
```

Comment:
what about older models? Are they not supported by the responses API?

https://github.com/vercel/ai/blob/7d739222e026a54ed36f0d2de6f0f026ecde631b/packages/gateway/src/gateway-language-model-settings.ts#L106-L115